### PR TITLE
I1499 refactor actions subcard 4

### PR DESCRIPTION
**Pull request by Serhiy91**
Deleted old Action components and mutations

### CODE REVIEW

File: app/imports/client/react/actions/components/ActionCompletionForm.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ActionCompletionForm = ({
                   placeholderText="Completion - target date"
                   onChange={save}
                   render={DatePickerAdapter}
-                  disabled={isCompleted || false}
+                  disabled={isVerified || false}
```
Comment (Dartv) [inline app/imports/client/react/actions/components/ActionCompletionForm.js]: Why? Original one had `isCompleted`. Also, why is there `false`?

File: app/imports/client/react/actions/containers/ActionAddContainer.js [hunk 2]
```diff
@@ -48,8 +49,15 @@ const ActionAddContainer = ({
...
-      linkedTo: linkedToField,
-      initialValues: getActionFormInitialState(user),
+      linkedTo,
+      initialValues: {
+        active: 0,
+        owner: getUserOptions(user),
+        toBeCompletedBy: getUserOptions(user),
+        planInPlace: ActionPlanOptions.NO,
+        // TODO: Update based on linked documents like creation modal?
+        completionTargetDate: moment().add(1, 'days'),
```
Comment (Dartv) [inline app/imports/client/react/actions/containers/ActionAddContainer.js]: dates should be plain js objects
```suggestion
        completionTargetDate: moment().add(1, 'days').toDate();
```

**Comment by Dartv** [comment]
I can't test this until you fix conflicts with main branch

File: app/imports/client/react/actions/components/ActionCompletionForm.js (hunk 1 above)
Comment (Serhiy91) [reply app/imports/client/react/actions/components/ActionCompletionForm.js]: Or do not render empty form while fetching. Can we do something else?

**Comment by Serhiy91** [comment]
@Dartv Which is the branch? I don't see conflicts.

**Comment by Dartv** [comment]
i1177-release-step-5 branch

**Comment by Serhiy91** [comment]
@Dartv I don't see conflicts.

**Pull request merged**